Exercise_A/index.html

```diff
@@ -10,7 +10,15 @@
   </head>
   <body>
     <!-- CODE HERE -->
-
+    <a href="about.html">
+      <img
+        class="card-img-top"
```

Comment:
When adding an image, the image should be inside the folder 
Then you would need to also add in `./images_A/Dnd_.jpg` this will then select the image